float32 voltage
float32 current
float32 charge
#float32 capacity
#float32 design_capacity
float32 percentage
uint8 power_supply_status
uint8 power_supply_health
#uint8 power_supply_technology
bool present
---
bool state